src/main/java/serverutils/integration/vp/VPIntegration.java

```diff
@@ -24,7 +27,23 @@ public static void init() {
     public static void updateMap(MessageJourneyMapUpdate message) {
         for (ClientClaimedChunks.Team team : message.teams.values()) {
             CLAIMS.putAll(team.chunkPos);
+            if (OWNTEAM == null && team.isMember) {
+                OWNTEAM = team.chunkPos.values().iterator().next().copy().setLoaded(false);
```

Comment:
Is it possible for `team.chunkPos` map to be empty at this point (when unloading or removing claim from last chunk)?